cypress/integration/inventory/export.spec.js

```diff
@@ -0,0 +1,37 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import InventoryActions from '../../support/fragments/inventory/inventoryActions';
+import FileManager from '../../support/utils/fileManager';
+
+
+describe('inventory / actions: export UUIDs', () => {
+  beforeEach('navigates to Inventory', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  it('C9284 verifies export UUIDs instances', () => {
+    cy.do(InventorySearch.searchByEffectiveLocation());
+    InventoryActions.saveUUIDs();
```

Comment:
This is a bare statement, and should be part of the `cy.do`, no?

Reply:
you right, fixed